src/Utilities/ProjectData.cpp

```diff
@@ -14,28 +14,25 @@
 
 
 #include "ProjectData.h"
+#include <string_view>
 #include "Message/Communicate.h"
 #include "Host/sysutil.h"
 #include "Utilities/qmc_common.h"
 #include "OhmmsData/ParameterSet.h"
+#include "Message/UniformCommunicateError.h"
 
 namespace qmcplusplus
 {
 //----------------------------------------------------------------------------
 // ProjectData
 //----------------------------------------------------------------------------
 // constructors and destructors
-ProjectData::ProjectData(const char* aname)
-    : m_title("asample"), m_host("none"), m_date("none"), m_series(0), m_cur(NULL), max_cpu_secs_(360000)
+ProjectData::ProjectData(const std::string& driver_epoch)
+    : m_host("none"), m_date("none"), m_series(0), m_cur(NULL), max_cpu_secs_(360000), driver_epoch_(driver_epoch)
 {
-  myComm = OHMMS::Controller;
-  if (aname == 0)
-  {
-    m_title = getDateAndTime("%Y%m%dT%H%M");
-    setName(m_title);
-  }
-  else
-    setName(aname);
+  myComm  = OHMMS::Controller;
+  m_title = getDateAndTime("%Y%m%dT%H%M");
```

Comment:
This is a breaking change. Please restore setting m_title via constructor. We need a way to test title without pulling xml.

Reply:
The way I see it you could never set the title.  It was just "asample" if you set aname.  This was never done in the code.  ProjectData doesn't need a name its type isn't polymorphic.

Reply:
It was never done in the main code but useful for unit testing. Just like the constructor argument of driver_epoch.